website/src/pages/product-ecosystem.module.css

```diff
@@ -1,3 +1,190 @@
+/* ROADMAP CSS */
+
+.roadmapSection {
+  flex-direction: column;
+  gap: 50px;
+}
+
+.roadmapContainer {
+  width: 90%;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  gap: 16px;
+}
+
+.roadmapColumn {
+}
+
+.roadmapColumnTitle {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  font-size: 1.2em;
+  font-weight: 700;
+  color: #555555;
+  text-align: center;
+  margin-bottom: 10px;
+}
+
+.titleIcon {
+  width: 16px;
+  vertical-align: sub;
+  margin-right: 5px;
+  color: grey;
+}
+
+.offCenterBalance {
+  width: 16px;
+}
+
+.roadmapItemContainer {
+  display: flex;
+  flex-direction: column;
+}
+
+.roadmapItemContainer .itemBtn {
+  width: 100%;
+  font-size: 0.85em;
+  font-weight: 500;
+  font-family: var(--aa-font-family);
+  text-align: left;
+  border: 1px solid rgb(205, 205, 205);
+  border-radius: 4px;
+  background-color: white;
+  padding: 10px 15px;
+  margin: 3px 0px;
+  border-left: 5px solid rgb(202, 202, 202);
+  box-shadow: 0px 2px 3px #c7c7c740;
+  cursor: pointer;
+  transition: background-color 150ms ease, border-color 150ms ease;
+}
+.roadmapItemContainer .itemBtn:hover {
+  background-color: rgb(240, 240, 240);
+  border-top-color: rgb(180, 180, 180);
+  border-bottom-color: rgb(180, 180, 180);
+  border-right-color: rgb(180, 180, 180);
+}
+
+.live .itemBtn {
+  border-left-color: rgb(84, 207, 92);
+}
+.comingSoon .itemBtn {
+  border-left-color: #4cabe7;
+}
+.backlog .itemBtn {
+  border-left-color: rgb(205, 183, 233); /* #92d8c3; */
+}
+.productsOnRadar .itemBtn {
+  border-left-color: rgb(240, 193, 127);
+}
+
+/* override ReactTooltip styling with custom */
+.tooltipStyling {
+  transition: opacity 150ms ease-out !important;
+  background: #263238 !important;
+  color: white !important;
+  box-shadow: 0px 3px 10px rgba(138, 138, 138, 0.602);
+}
+.tooltipStyling::after {
+  background-color: #263238 !important;
+}
+/***/
+
+.tooltipContent {
+  max-width: 300px;
+}
+
+.tooltipTitle {
+  font-weight: bold;
+}
+.tooltipDescription {
+  line-height: 1.4em;
+  margin-top: 1px;
+}
+.tooltipSmallText {
+  margin: 6px 0;
+  display: block;
+  color: #ccc;
+}
+
+/* fix margin between list items */
+.scrollbar > div {
+  display: flex;
+  flex-direction: column;
+  padding-bottom: 6px;
+}
+
+.scrollbar {
+  border-bottom: 2px solid #cfcfcf95;
+  padding-bottom: 1px;
+}
+
+.modalDialog {
+  min-width: 400px;
```

Comment:
We need to clear this on mobile width.

https://user-images.githubusercontent.com/1413275/209096762-f753a0c8-ba1a-4129-aa20-f8187c38796e.mov